src/musiccast-zone.ts

```diff
@@ -137,11 +137,6 @@ export class MusiccastZone {
             await McDeviceApi.setInput(this._device.ip, input as McInputId, this.zoneId);
         } else {
             this.log.warn('unkown input "{input}" for device {deviceid} in zone {zone}', input, this._device.id, this.zoneId);
-            if (this.useInputFriendlyNames) {
```

Comment:
Is there any reason why you delete these lines? I think it can be very useful to see the possible values.